{"abstract":[{"type":"text","text":"additional context information related to the error"}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/sotocore\/awsresponseerror\/context"]}],"metadata":{"modules":[{"name":"SotoCore"}],"roleHeading":"Instance Property","externalID":"s:8SotoCore16AWSResponseErrorV7contextAA15AWSErrorContextVSgvp","symbolKind":"property","role":"symbol","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"kind":"text","text":": "},{"text":"AWSErrorContext","kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"title":"context"},"sections":[],"primaryContentSections":[{"declarations":[{"tokens":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","text":"AWSErrorContext","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext"},{"text":"?","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError/context":{"fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8SotoCore15AWSErrorContextV","text":"AWSErrorContext","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"kind":"symbol","type":"topic","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","abstract":[{"text":"additional context information related to the error","type":"text"}],"url":"\/documentation\/sotocore\/awsresponseerror\/context","role":"symbol","title":"context"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError":{"title":"AWSResponseError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSResponseError"}],"abstract":[{"type":"text","text":"Response Error type returned by Soto if the error code is unrecognised"}],"type":"topic","kind":"symbol","url":"\/documentation\/sotocore\/awsresponseerror","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSResponseError"}]},"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","kind":"symbol","url":"\/documentation\/sotocore","abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}],"role":"collection","title":"SotoCore"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"type":"topic","url":"\/documentation\/sotocore\/awserrorcontext","title":"AWSErrorContext","kind":"symbol","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"AWSErrorContext","kind":"identifier"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","abstract":[{"type":"text","text":"Additional information about error"}]}}}